ts/src/pro/bybit.ts

```diff
@@ -938,6 +942,149 @@ export default class bybit extends bybitRest {
         client.resolve (trades, messageHash);
     }
 
+    async watchPositions (symbols: string[] = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name bybit#watchPositions
+         * @see https://bybit-exchange.github.io/docs/v5/websocket/private/position
+         * @description watch all open positions
+         * @param {[string]|undefined} symbols list of unified market symbols
+         * @param {object} params extra parameters specific to the bybit api endpoint
+         * @returns {[object]} a list of [position structure]{@link https://docs.ccxt.com/en/latest/manual.html#position-structure}
+         */
+        await this.loadMarkets ();
```

Comment:
@pcriadoperez again it is not returning the position after calling it the first time 
![image](https://github.com/ccxt/ccxt/assets/43336371/65b10d01-00a6-4c66-ba2d-fece4336b4e0)

Reply:
same with bitget
![image](https://github.com/ccxt/ccxt/assets/43336371/89a7feee-71cc-4283-931a-6bc76cf34cc6)

Reply:
I believe now it should be working
```
watchPositions Bitget
Array
(
    [0] => Array
        (
            [info] => Array
                (
                    [posId] => 926036334386778112
                    [instId] => LTCUSDT_UMCBL
                    [instName] => LTCUSDT
                    [marginCoin] => USDT
                    [margin] => 57.168
                    [marginMode] => crossed
                    [holdSide] => long
                    [holdMode] => double_hold
                    [total] => 1.8
                    [available] => 1.8
                    [locked] => 0
                    [averageOpenPrice] => 63.52
                    [leverage] => 2
                    [achievedProfits] => 0
                    [upl] => -0.756
                    [uplRate] => -0.0132
                    [liqPx] => -152.07
                    [keepMarginRate] => 0.010
                    [marginRate] => 0.020929977604
                    [cTime] => 1656510187717
                    [uTime] => 1695601537747
                    [markPrice] => 63.1
                    [autoMargin] => off
                )

            [id] => 926036334386778112
            [symbol] => LTC/USDT:USDT
            [notional] => 
            [marginMode] => cross
            [liquidationPrice] => 
            [entryPrice] => 63.52
            [unrealizedPnl] => -0.756
            [percentage] => -0.0132
            [contracts] => 1.8
            [contractSize] => 1
            [markPrice] => 63.1
            [side] => long
            [hedged] => 1
            [timestamp] => 1695601537747
            [datetime] => 2023-09-25T00:25:37.747Z
            [maintenanceMargin] => 
            [maintenanceMarginPercentage] => 0.01
            [collateral] => 
            [initialMargin] => 
            [initialMarginPercentage] => 
            [leverage] => 2
            [marginRatio] => 0.020929977604
        )

    [1] => Array
        (
            [info] => Array
                (
                    [posId] => 926036334386778114
                    [instId] => LTCUSDT_UMCBL
                    [instName] => LTCUSDT
                    [marginCoin] => USDT
                    [margin] => 47.8695
                    [marginMode] => crossed
                    [holdSide] => short
                    [holdMode] => double_hold
                    [total] => 1.5
                    [available] => 1.5
                    [locked] => 0
                    [averageOpenPrice] => 63.82
                    [leverage] => 2
                    [achievedProfits] => 0
                    [upl] => 1.0889
                    [uplRate] => 0.0227
                    [liqPx] => -152.07
                    [keepMarginRate] => 0.010
                    [marginRate] => 0.020929977604
                    [cTime] => 1656510187717
                    [uTime] => 1695601094998
                    [markPrice] => 63.1
                    [autoMargin] => off
                )

            [id] => 926036334386778114
            [symbol] => LTC/USDT:USDT
            [notional] => 
            [marginMode] => cross
            [liquidationPrice] => 
            [entryPrice] => 63.82
            [unrealizedPnl] => 1.0889
            [percentage] => 0.0227
            [contracts] => 1.5
            [contractSize] => 1
            [markPrice] => 63.1
            [side] => short
            [hedged] => 1
            [timestamp] => 1695601094998
            [datetime] => 2023-09-25T00:18:14.998Z
            [maintenanceMargin] => 
            [maintenanceMarginPercentage] => 0.01
            [collateral] => 
            [initialMargin] => 
            [initialMarginPercentage] => 
            [leverage] => 2
            [marginRatio] => 0.020929977604
        )

    [2] => Array
        (
            [info] => Array
                (
                    [posId] => 1080971971354304526
                    [instId] => ETHUSDT_UMCBL
                    [instName] => ETHUSDT
                    [marginCoin] => USDT
                    [margin] => 1.6994
                    [marginMode] => crossed
                    [holdSide] => short
                    [holdMode] => double_hold
                    [total] => 0.02
                    [available] => 0.02
                    [locked] => 0
                    [averageOpenPrice] => 1699.46
                    [leverage] => 20
                    [achievedProfits] => 0
                    [upl] => 2.5573
                    [uplRate] => 1.5047
                    [liqPx] => 0
                    [keepMarginRate] => 0.005
                    [marginRate] => 0.020929977604
                    [cTime] => 1693449722135
                    [uTime] => 1693450706210
                    [markPrice] => 1571.6
                    [autoMargin] => off
                )

            [id] => 1080971971354304526
            [symbol] => ETH/USDT:USDT
            [notional] => 
            [marginMode] => cross
            [liquidationPrice] => 
            [entryPrice] => 1699.46
            [unrealizedPnl] => 2.5573
            [percentage] => 1.5047
            [contracts] => 0.02
            [contractSize] => 1
            [markPrice] => 1571.6
            [side] => short
            [hedged] => 1
            [timestamp] => 1693450706210
            [datetime] => 2023-08-31T02:58:26.210Z
            [maintenanceMargin] => 
            [maintenanceMarginPercentage] => 0.005
            [collateral] => 
            [initialMargin] => 
            [initialMarginPercentage] => 
            [leverage] => 20
            [marginRatio] => 0.020929977604
        )

    [3] => Array
        (
            [info] => Array
                (
                    [posId] => 1080971971354304528
                    [instId] => ETHUSDT_UMCBL
                    [instName] => ETHUSDT
                    [marginCoin] => USDT
                    [margin] => 1.7006
                    [marginMode] => crossed
                    [holdSide] => long
                    [holdMode] => double_hold
                    [total] => 0.02
                    [available] => 0.02
                    [locked] => 0
                    [averageOpenPrice] => 1700.62
                    [leverage] => 20
                    [achievedProfits] => 0
                    [upl] => -2.5804
                    [uplRate] => -1.5173
                    [liqPx] => 0
                    [keepMarginRate] => 0.005
                    [marginRate] => 0.020929977604
                    [cTime] => 1693449722135
                    [uTime] => 1693459050488
                    [markPrice] => 1571.6
                    [autoMargin] => off
                )

            [id] => 1080971971354304528
            [symbol] => ETH/USDT:USDT
            [notional] => 
            [marginMode] => cross
            [liquidationPrice] => 
            [entryPrice] => 1700.62
            [unrealizedPnl] => -2.5804
            [percentage] => -1.5173
            [contracts] => 0.02
            [contractSize] => 1
            [markPrice] => 1571.6
            [side] => long
            [hedged] => 1
            [timestamp] => 1693459050488
            [datetime] => 2023-08-31T05:17:30.488Z
            [maintenanceMargin] => 
            [maintenanceMarginPercentage] => 0.005
            [collateral] => 
            [initialMargin] => 
            [initialMarginPercentage] => 
            [leverage] => 20
            [marginRatio] => 0.020929977604
        )

)
```

Reply:
bybit
```php
bybit->watchPositions()
Array
(
    [0] => Array
        (
            [info] => Array
                (
                    [positionIdx] => 0
                    [riskId] => 1
                    [riskLimitValue] => 2000000
                    [symbol] => BTCUSDT
                    [side] => Sell
                    [size] => 0.012
                    [avgPrice] => 26557.39166667
                    [positionValue] => 318.6887
                    [tradeMode] => 0
                    [positionStatus] => Normal
                    [autoAddMargin] => 0
                    [adlRankIndicator] => 2
                    [leverage] => 10
                    [positionBalance] => 32.06167667
                    [markPrice] => 26131.60
                    [liqPrice] => 
                    [bustPrice] => 199999.80
                    [positionMM] => 11.999988
                    [positionIM] => 3.186887
                    [tpslMode] => Full
                    [takeProfit] => 0.00
                    [stopLoss] => 0.00
                    [trailingStop] => 0.00
                    [unrealisedPnl] => 5.1095
                    [cumRealisedPnl] => -0.18784982
                    [seq] => 8179853146
                    [createdTime] => 1695566950161
                    [updatedTime] => 1695601716039
                )

            [id] => 
            [symbol] => BTC/USDT:USDT
            [timestamp] => 1695601716039
            [datetime] => 2023-09-25T00:28:36.039Z
            [lastUpdateTimestamp] => 
            [initialMargin] => 3.186887
            [initialMarginPercentage] => 0.01
            [maintenanceMargin] => 11.999988
            [maintenanceMarginPercentage] => 0.037654262607993
            [entryPrice] => 26557.39166667
            [notional] => 318.6887
            [leverage] => 10
            [unrealizedPnl] => 5.1095
            [contracts] => 0.012
            [contractSize] => 1
            [marginRatio] => 0.3742
            [liquidationPrice] => 
            [markPrice] => 26131.6
            [lastPrice] => 
            [collateral] => 32.06167667
            [marginMode] => cross
            [side] => short
            [percentage] => 
            [stopLossPrice] => 0
            [takeProfitPrice] => 0
        )

    [1] => Array
        (
            [info] => Array
                (
                    [positionIdx] => 0
                    [riskId] => 71
                    [riskLimitValue] => 200000
                    [symbol] => LTCUSDT
                    [side] => Buy
                    [size] => 0.1
                    [avgPrice] => 63.2
                    [positionValue] => 6.32
                    [tradeMode] => 0
                    [positionStatus] => Normal
                    [autoAddMargin] => 0
                    [adlRankIndicator] => 2
                    [leverage] => 10
                    [positionBalance] => 0.6351284
                    [markPrice] => 63.20
                    [liqPrice] => 
                    [bustPrice] => 0.01
                    [positionMM] => 0.00001
                    [positionIM] => 0.1264
                    [tpslMode] => Full
                    [takeProfit] => 0.00
                    [stopLoss] => 0.00
                    [trailingStop] => 0.00
                    [unrealisedPnl] => 0
                    [cumRealisedPnl] => -0.003476
                    [seq] => 15216554353
                    [createdTime] => 1695566950162
                    [updatedTime] => 1695601716039
                )

            [id] => 
            [symbol] => LTC/USDT:USDT
            [timestamp] => 1695601716039
            [datetime] => 2023-09-25T00:28:36.039Z
            [lastUpdateTimestamp] => 
            [initialMargin] => 0.1264
            [initialMarginPercentage] => 0.02
            [maintenanceMargin] => 1.0E-5
            [maintenanceMarginPercentage] => 1.582278481012E-6
            [entryPrice] => 63.2
            [notional] => 6.32
            [leverage] => 10
            [unrealizedPnl] => 
            [contracts] => 0.1
            [contractSize] => 1
            [marginRatio] => 0
            [liquidationPrice] => 
            [markPrice] => 63.2
            [lastPrice] => 
            [collateral] => 0.6351284
            [marginMode] => cross
            [side] => long
            [percentage] => 
            [stopLossPrice] => 0
            [takeProfitPrice] => 0
        )

)
```